pages/add-clinic.vue

```diff
@@ -132,10 +143,26 @@ export default {
             .firestore()
             .collection("pending")
             .add(clinic)
-            .then(() => console.log("added to db", this.clinic))
-            .then(this.$router.push("/"));
+            .then(() => {
+              console.log("added to db", this.clinic)
+              this.snackBarData = {
+                open: true,
+                text: "Added Successfully to the database",
+                onClosed: () => {
+                  this.snackBarData.open = false;
+                  this.$router.push("/")
```

Comment:
this shouldn't be tied to the close event since we can't know when it was. 
Move this to after the snackbar is opened and add a delay  like this:

```
await new Promise((resolve) => setTimeout(resolve, 1500));
```

Note: `submitData()` will need to be `async`